tests/src/test/scala/org/http4s/internal/RandomSuite.scala

```diff
@@ -0,0 +1,68 @@
+package org.http4s.internal
+
+import cats.effect.IO
+import org.http4s.Http4sSuite
+
+import java.security.SecureRandom
+import java.util.{Random => JRandom}
+import scala.util.Try
+
+class RandomSuite extends Http4sSuite {
+  test("javaUtilRandomBlocking.nextBigInt works") {
+    val rnd = new JRandom()
+    Random
+      .javaUtilRandomBlocking[IO](testBlocker)(rnd)
+      .nextBigInt(10)
+      .map(_ > 0)
```

Comment:
Why `_ > 0`?

https://docs.oracle.com/javase/8/docs/api/java/math/BigInteger.html#BigInteger-int-java.util.Random-

Reply:
Because my son's hockey practice was almost over and I forgot what I was doing.  The tests could be property tests.

Reply:
It's now a property test.